preprocessing_Iteration_1.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Mar 30 09:28:58 2019
+
+@author: moealani
+"""
+
+
+import glob
+import cv2
+import os
+import numpy as np
+
+#### PART 1: reading images from a folder ##### function to load an image from a folder
+
+
+## the stored_images_path parameter should be changed to wherever your images are stored
+
+stored_images_path = [img for img in glob.glob("/Users/moealani/PycharmProjects/MSR/data/*.png")]
```

Comment:
I was thinking about at least setting up the project directory structure so that we extract into the Data folder. Then we could use a relative path on all our machines I think?